src/Microsoft.DotNet.Interactive.Tests/LanguageServices/SignatureHelpTests.cs

```diff
@@ -115,5 +116,34 @@ public async Task signature_help_can_return_doc_comments_from_source(Language la
                 .Should()
                 .Be(expectedMethodDocumentation);
         }
+
+        [Theory]
+        [InlineData(Language.CSharp, "new SampleLibrary.SampleLibrary($$")]
+        //[InlineData(Language.FSharp, "let s = SampleLibrary.SampleLibrary($$")] // F# doesn't support signature help yet
```

Comment:
Please uncomment this and put a `Skip` on it so it shows up in test runs as skipped. It's a little easier to keep track of that way.

Reply:
The `Skip` attribute isn't properly honored by VS test discovery, so this breaks my workflow.  Issue [here](https://github.com/xunit/xunit/issues/1782).

Reply:
I've removed the F# test since it's known to not yet work via #1031.